api/src/AppBundle/Controller/UserController.php

```diff
@@ -251,17 +254,20 @@ public function getUserTeamNames(Request $request, $email)
      * //TODO move to UserService
      *
      * @Route("/{id}", methods={"DELETE"})
-     * @Security("has_role('ROLE_ADMIN')")
+     * @Security("has_role('ROLE_ADMIN') or has_role('ROLE_SUPER_ADMIN')")
```

Comment:
I think roles are configured hierarchically, so just `has_role('ROLE_ADMIN')` will implicitly include `ROLE_SUPER_ADMIN`.

Reply:
Awesome - I'll drop it.